src/message/messagesActions.js

```diff
@@ -17,11 +17,13 @@ export const switchNarrow = (narrow: Narrow) => ({
   narrow,
 });
 
-export const doNarrow = (newNarrow: Narrow, anchor: number = Number.MAX_SAFE_INTEGER) =>
-  (dispatch: Dispatch, getState: GetState) => {
-    registerAppActivity(getAuth(getState()));
-    requestIdleCallback(() => dispatch(switchNarrow(newNarrow)));
-  };
+export const doNarrow = (newNarrow: Narrow, anchor: number = Number.MAX_SAFE_INTEGER) => (
```

Comment:
Same for formatting here. Have a look at Prettier.js which we should plan on using by default soon?!?